src/smc-hub/postgres.coffee

```diff
@@ -1,28 +1,30 @@
 ###
 PostgreSQL database entry point.
+Do not import any of the submodules directly unless you
+know exactly what you're doing.
 
 COPYRIGHT : (c) 2017 SageMath, Inc.
 LICENSE   : AGPLv3
 ###
-require('coffee-cache')
+require('coffee2-cache')
 
 fs           = require('fs')
 
 base = require('./postgres-base')
 for f in ['pg_type', 'expire_time', 'one_result', 'all_results', 'count_result']
     exports[f] = base[f]
 
-exports.PUBLIC_PROJECT_COLUMNS = ['project_id',  'last_edited', 'title', 'description', 'deleted',  'created']
-exports.PROJECT_COLUMNS = ['users'].concat(exports.PUBLIC_PROJECT_COLUMNS)
-
+exports.PUBLIC_PROJECT_COLUMNS = base.PUBLIC_PROJECT_COLUMNS
+exports.PROJECT_COLUMNS        = base.PROJECT_COLUMNS
 
 # Add further functionality to PostgreSQL class -- must be at the bottom of this file.
-# Each of the following calls extends the PostgreSQL class with further important functionality.
+# Each of the following calls composes the PostgreSQL class with further important functionality.
```

Comment:
got to say that changing extends to composes is inconsistent with `extend_PostgreSQL` below...